assets/wizards/site-design/views/main/index.js

```diff
@@ -0,0 +1,275 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+import { alignCenter, alignLeft } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	withWizardScreen,
+	ColorPicker,
+	TextControl,
+	SelectControl,
+	ToggleControl,
+	SectionHeader,
+	ImageUpload,
+	hooks,
+	Button,
+	Grid,
+	WebPreview,
+} from '../../../../components/src';
+import ThemeSelection from '../../components/theme-selection';
+import { getFontsList, getFontImportURL, LOGO_SIZE_OPTIONS, parseLogoSize } from './utils';
+import './style.scss';
+
+const Main = ( {
+	wizardApiFetch,
+	setError,
+	renderPrimaryButton,
+	buttonText,
+	hasPreview = true,
+} ) => {
+	const [ themeSlug, updateThemeSlug ] = useState();
+	const [ mods, updateMods ] = hooks.useObjectState();
+
+	const updateSettings = response => {
+		updateMods( response.theme_mods );
+		updateThemeSlug( response.theme );
+	};
+	useEffect( () => {
+		wizardApiFetch( {
+			path: '/newspack/v1/wizard/newspack-setup-wizard/theme',
+		} )
+			.then( updateSettings )
+			.catch( setError );
+	}, [] );
+
+	const saveSettings = () =>
+		wizardApiFetch( {
+			path: '/newspack/v1/wizard/newspack-setup-wizard/theme/',
+			method: 'POST',
+			data: { theme_mods: mods, theme: themeSlug },
+			quiet: true,
+		} )
+			.then( updateSettings )
+			.catch( setError );
+
+	return (
+		<>
+			<SectionHeader
+				title={ __( 'Theme', 'newspack' ) }
+				description={ __( 'Activate a theme you like to get started', 'newspack' ) }
+			/>
+			<ThemeSelection theme={ themeSlug } updateTheme={ updateThemeSlug } />
+			<SectionHeader
+				title={ __( 'Colors', 'newspack' ) }
+				description={ __( 'Define your primary and secondary colors', 'newspack' ) }
+			/>
+			<Grid gutter={ 32 }>
```

Comment:
This UI is a bit confusing to me. In the Customizer, we at first have two options, "Default" and "Custom":

<img width="459" alt="Screen Shot 2021-03-05 at 2 05 30 PM" src="https://user-images.githubusercontent.com/2230142/110174492-6dd68b80-7dbd-11eb-8cff-fc5b1b8a5add.png">

Only when you select "Custom" do you see the color pickers.

In the new Onboarding and Wizard screens, the default behavior seems to be the same as "Default", but the color pickers both show `#fff`, which doesn't actually seem to be the case. Maybe we should have a toggle control to set "Default" vs. "Custom", and only show the pickers if the control is set to Custom? This would mirror the Customizer and make the default state a bit more understandable, IMO.

Reply:
> the default behavior seems to be the same as "Default", but the color pickers both show #fff, which doesn't actually seem to be the case. 

How to reproduce that? When I set the colors to "Default" in the customizer, I can still see the primary & secondary colors in the  Onboarding and Wizard screens.

> Maybe we should have a toggle control to set "Default" vs. "Custom", and only show the pickers if the control is set to Custom? 

The idea here was to simplify color picking - instead of default vs custom there's just a color picker.

Reply:
Update the colors logic to handle the theme's custom vs. default colors in 7dcf5c5ddd90b24eb4cd859777c197718f3b782f - maybe that resolves the issue

Reply:
> How to reproduce that?

Sorry I didn't get to posting replication steps. I can say that 7dcf5c5 fixes the issue partially for me, but the suggestion I made below fixes it completely. To test:

1. In the Customizer, choose "Custom Colors" and set a hex value for primary and secondary colors
2. In the Site Design page, observe that the primary color is shown, but the secondary shows `#fff`
3. Apply the suggested patch and refresh Site Design. Confirm that both custom colors are shown.
4. In the Customizer, reset the colors back to "Default."
5. In the Site Design page, confirm that the colors shown are now the theme's default colors.

Reply:
Ah, that makes sense! Thanks, added in 0f75d5b36196dcf0225e13d83a53cfc7bdd5b586